scripts/mediaViews.php

```diff
@@ -11,19 +11,25 @@
 $db = Doctrine_Manager::getInstance()->getCurrentConnection();
 
 //prune logs
-$date = date('Y-m-d H:i:s', strtotime('1 month ago'));
-
-$sql = "DELETE FROM media_views WHERE datecreated < ?";
+$sql = "DELETE FROM media_views WHERE datecreated < (NOW() - INTERVAL 1 MONTH)";
```

Comment:
Ah I see. It's a temporary data table from whence the actual counts of popularity and total views are updated on the media table.